### PR TITLE
Fix point highlight and console error

### PR DESCRIPTION
**Pull request by RenataMuellerC2C**
Fix: GSGMF-1499

### CODE REVIEW

File: contribs/gmf/src/editing/editFeatureComponent.js [hunk 1]
```diff
@@ -563,6 +563,12 @@ Controller.prototype.$onInit = function () {
     throw new Error('Missing map');
   }
 
+  this.options_.highlightStyle.circle = {
+    radius: 7,
+    fill: this.options_.highlightStyle.fill,
+    stroke: this.options_.highlightStyle.stroke,
+  };
```
Comment (sbrunner) [inline contribs/gmf/src/editing/editFeatureComponent.js:570]: Why this?
Looks like that it should be moved here: https://github.com/camptocamp/c2cgeoportal/blob/2.6/geoportal/c2cgeoportal_geoportal/scaffolds/update/geoportal/CONST_vars.yaml_tmpl#L319

**Review by sbrunner — COMMENTED** [review]
Looks good :-)

File: contribs/gmf/src/options.js [hunk 2]
```diff
@@ -465,7 +465,7 @@
 
 /**
  * @typedef {Object} gmfEditFeatureOptions
- * @property {import('ngeo/options.js').StyleLike} highlightStyle A style object for highlighting of feature.
+ * @property {import('ngeo/options.js').Style} highlightStyle A style object for highlighting of feature.
```
Comment (llienher) [inline contribs/gmf/src/options.js:468]: What is this change for ? OL use StyleLike (https://openlayers.org/en/latest/apidoc/module-ol_style_Style.html#~StyleLike) and there is 9 others occurance of it in this same file.

Comment (sbrunner) [reply contribs/gmf/src/options.js:468]: +1 it should be `StyleLike`

**Review by llienher — APPROVED** [review]
Looks good, thanks !

**Pull request merged**